OWNERS_ALIASES

```diff
@@ -31,39 +30,29 @@ aliases:
   - enxebre
 
   # -----------------------------------------------------------
-  # OWNER_ALIASES for docs/book
+  # OWNER_ALIASES for bootstrap/kubeadm
   # -----------------------------------------------------------
 
-  # folks who can review and LGTM any PRs under docs/book
-  cluster-api-book-reviewers:
-  - moshloop
-  - randomvariable
+  cluster-api-bootstrap-provider-kubeadm-maintainers:
```

Comment:
From here onward these groups are empty, keeping them here as we grow could be useful to add more specialized folks

Reply:
+1, we should follow up on adding specialized reviewers 

if anyone is interested in a particular area, let us know!